map/set-max-extent/src/main/java/com.esri.samples.set_max_extent/SetMaxExtentSample.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2022 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.set_max_extent;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.ToggleButton;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Color;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.symbology.*;
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.BasemapStyle;
+import com.esri.arcgisruntime.mapping.view.MapView;
+
+public class SetMaxExtentSample extends Application {
+
+    private MapView mapView;
+
+
+    @Override
+    public void start(Stage stage) {
+
+        try {
+            // create stack pane and application scene
+            var stackPane = new StackPane();
+            var scene = new Scene(stackPane);
+
+            // set title, size, and add scene to stage
+            stage.setTitle("Set Max Extent Sample");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // authentication with an API key or named user is required to access basemaps and other location services
+            String yourAPIKey = System.getProperty("apiKey");
+            ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+            // create a map with the streets basemap style
+            var map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
```

Comment:
I tend to declare the explicit type with `ArcGISMap`. 

`map` can have many meanings in Java, and for Java developers, a geographical map may not necessarily be the first thing they think of when seeing a variable name later down the code! So for the avoidance of all doubt, we keep `ArcGISMap` declared and avoid the use of var here.

Another option would be to name the variable `arcGISMap` but it's not quite as readable as map later down the code. 

```suggestion
            ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
```